source/MasterData.Integrationtests/Fixture/MasterDataIntegrationTestCollection.cs

```diff
@@ -12,13 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using System;
+using Xunit;
 
-namespace Energinet.DataHub.EDI.IntegrationTests.Application.Actors;
+namespace Energinet.DataHub.EDI.MasterData.IntegrationTests.Fixture;
 
-internal static class SampleData
+[CollectionDefinition(nameof(MasterDataTestCollectionFixture))]
+public class MasterDataTestCollectionFixture : ICollectionFixture<MasterDataFixture>
```

Comment:
```suggestion
public class MasterDataTestCollection : ICollectionFixture<MasterDataFixture>
```
Rename since it is a collection, not a fixture. Else the file should be renamed to match the class name.